ui/components/app/confirm/info/row/url.tsx

```diff
@@ -16,13 +16,32 @@ import {
   TextVariant,
   BackgroundColor,
 } from '../../../../../helpers/constants/design-system';
+import SnapAuthorshipPill from '../../../snaps/snap-authorship-pill';
+import { SnapMetadataModal } from '../../../snaps/snap-metadata-modal';
+import { isSnapId } from '../../../../../helpers/utils/snaps';
 
 export type ConfirmInfoRowUrlProps = {
   url: string;
 };
 
 export const ConfirmInfoRowUrl = ({ url }: ConfirmInfoRowUrlProps) => {
   let urlObject;
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const openModal = () => setIsModalOpen(true);
```

Comment:
Minor, is it worth `useCallback` on these?

Reply:
Yeah, that's fair. [e53f2d1](https://github.com/MetaMask/metamask-extension/pull/26881/commits/e53f2d14658f51b2ae302ea48df88dad07976621)